### PR TITLE
{bio}[GCCcore/8.2.0,foss/2019a] 3d-dna v180922, LASTZ v1.02.00 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/theaidenlab/%(name)s/archive']
+sources = ['%s.zip' % local_githash]
+checksums = ['348c3e019ea29e47382eb2d85228a56bc11b316c130afabae016ad8e7d7640ca']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('LASTZ', '1.02.00'),
+    ('Java', '1.8', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb]: Please use `Java/11` (like we already do in other easyconfigs using a `2019a` toolchain)

File: easybuild/easyconfigs/l/LASTZ/LASTZ-1.02.00-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS 
+ sequencing technologies such as Roche 454.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.bx.psu.edu/miller_lab/dist/']
+
+patches = ['LASTZ-%(version)s_Makefile.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/l/LASTZ/LASTZ-1.02.00-GCCcore-8.2.0.eb]: please add the missing SHA256 checksums (`eb --inject-checksums` is your friend here)

File: easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+name = '3d-dna'
+version = '180922'
+versionsuffix = '-Python-%(pyver)s'
+local_githash = '529ccf46599825b3047e58a69091d599e9858a19'
+
+homepage = 'https://github.com/theaidenlab/3d-dna'
+description = """3D de novo assembly (3D DNA) pipeline"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (boegel) [inline easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb]: Do you really need to use `foss` here, none of the dependencies require it?

Comment (timeu) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb]: Should this then be the SYSTEM toolchain or GCC or GCCCore ?

**Comment by verdurin** [comment]
@timeu Any updates here?

**Review by jfgrimm — COMMENTED** [review]
As decided in issue #16330, we have deprecated the use of `True` to signify a system-toolchain dependency (#16384), in favour of the more intuitive `SYSTEM` template constant. Due to the change in the test suite, please run `eb --sync-pr-with-develop ${pr}` and update the PR to use `SYSTEM` instead.

File: easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb:11]:
```suggestion
toolchain = {'name': 'GCC', 'version': '8.2.0'}

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/0/3d-dna/3d-dna-180922-GCCcore-8.2.0-Python-2.7.15.eb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/theaidenlab/%(name)s/archive']
+sources = ['%s.zip' % local_githash]
+checksums = ['348c3e019ea29e47382eb2d85228a56bc11b316c130afabae016ad8e7d7640ca']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('LASTZ', '1.02.00'),
+    ('Java', '1.11', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/0/3d-dna/3d-dna-180922-GCCcore-8.2.0-Python-2.7.15.eb:20]:
```suggestion
    ('Java', '11', '', True),

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/0/3d-dna/3d-dna-180922-GCCcore-8.2.0-Python-2.7.15.eb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/theaidenlab/%(name)s/archive']
+sources = ['%s.zip' % local_githash]
+checksums = ['348c3e019ea29e47382eb2d85228a56bc11b316c130afabae016ad8e7d7640ca']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('LASTZ', '1.02.00'),
+    ('Java', '11', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/0/3d-dna/3d-dna-180922-GCCcore-8.2.0-Python-2.7.15.eb:20]:
```suggestion
    ('Java', '11', '', SYSTEM),

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/0/3d-dna/3d-dna-180922-GCCcore-8.2.0-Python-2.7.15.eb [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/theaidenlab/%(name)s/archive']
+sources = ['%s.zip' % local_githash]
+checksums = ['348c3e019ea29e47382eb2d85228a56bc11b316c130afabae016ad8e7d7640ca']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('LASTZ', '1.02.00'),
+    ('Java', '11', '', SYSTEM),
```
Comment (boegel) [inline easybuild/easyconfigs/0/3d-dna/3d-dna-180922-GCCcore-8.2.0-Python-2.7.15.eb:20]:
```suggestion
    ('Java', '1.8', '', SYSTEM),
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @timeu!

**Pull request merged**